### PR TITLE
Homework2 - completed

### PR DESCRIPTION
**Pull request by Nikolaenko94**
@vitalics **HOMEWORK 2 IS DONE**

### CODE REVIEW

File: hw2/arrayfilter.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+//TASK 2.5
+function ageAndSexFilter(age,sex,array){
+    let ageAndSexFilter = array.filter(element => element.age === age && element.sex === sex);
+    return ageAndSexFilter;
+};
+
+//TASK 2.6
+const addNewStudent = (newStudent,array) => {
+    array.unshift(newStudent)
```
Comment (vitalics) [inline hw2/arrayfilter.js]: after using this function - we will change the initial array.

File: hw2/converter.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+globalThis.converter=function(arr){
```
Comment (vitalics) [inline hw2/converter.js:1]: Math complexity of this algorithm = O(5n). 
When Array Length will be 100 - we iterate 500 times

**Review by vitalics — CHANGES_REQUESTED** [review]

File: hw2/arrayfilter.js (hunk 1 above)
Comment (Nikolaenko94) [reply hw2/arrayfilter.js]: fixed

File: hw2/converter.js (hunk 2 above)
Comment (Nikolaenko94) [reply hw2/converter.js:1]: Fixed.
If I'm not mistaken, now math complexity of this algorithm = O(n) ...

**Comment by Nikolaenko94** [comment]
Fixed all remarks

File: hw2/index.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+// import 1 task
+require("./converter");
+// import 2 task (also imported array = students)
+let arrayFilter = require("./arrayfilter");
+    
+
```
Comment (vitalics) [inline hw2/index.js]: too many empty lines

**Review by vitalics — APPROVED** [review]

File: hw2/index.js (hunk 3 above)
Comment (Nikolaenko94) [reply hw2/index.js]: fixed

**Pull request merged**